doc/src/fix_ave_correlate.rst

```diff
@@ -257,6 +258,9 @@ time.
 * If *type* is set to *full* then each input value is correlated with
   itself and every other value (i.e., :math:`C_{ij} = V_i V_j` for
   :math:`\{i,j\} = \{1,N\}`, so :math:`N_\text{pair} = N^2`).
+* If *type* is set to *first* then each input value is correlated with
+  the first input value (i.e., :math:`C_{ij} = V_1 V_j` for
+  :math:`\{i,j\} = \{1,N\}`, so :math:`N_\text{pair} = N`).
```

Comment:
should this just be j = (1,N) since i is not varying ?

Reply:
Yes - my mistake! Missed it when I copy-pasted from type full. Can fix in an hour or so when I get to the office if you don't beat me to it.